docs/en/observability/tab-widgets/uptime-monitoring/set-up-docker-content.asciidoc

```diff
@@ -0,0 +1,31 @@
+// tag::heartbeat[]
+
+[source,sh,subs="attributes"]
+----
+docker pull docker.elastic.co/beats/heartbeat:{version}
+----
+
+// end::heartbeat[]
+
+// tag::agent[]
+
+There are two Docker images you can use based on the type of monitor you are setting up.
+
+For *browser checks*, you must use the `elastic-agent-complete` Docker image to create a self-hosted {agent} node.
+This image contains the dependencies to run synthetic monitors.
+The standard Elastic Cloud or self-hosted {agent} will not work.
+
+[source,sh,subs="attributes"]
+----
+docker pull docker.elastic.co/beats/elastic-agent-complete:{version}
+----
+
+For *lightweight HTTP/S, TCP, and ICMP checks*, you can use either monitor the
+`elastic-agent-complete` image mentioned above or the following `elastic-agent` image:
+
+[source,sh,subs="attributes"]
+----
+docker pull docker.elastic.co/beats/elastic-agent-complete:{version}
+----
```

Comment:
Not sure if this is true and/or something we would want to mention if it is... 

Also typo: 

```suggestion
For *lightweight HTTP/S, TCP, and ICMP checks*, you can use either monitor the
`elastic-agent-complete` image mentioned above or the following `elastic-agent` image:

[source,sh,subs="attributes"]
----
docker pull docker.elastic.co/beats/elastic-agent:{version}
----
```